src/main/java/tech/ypsilon/bbbot/database/structs/VerificationDocument.java

```diff
@@ -0,0 +1,18 @@
+package tech.ypsilon.bbbot.database.structs;
+
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.bson.types.ObjectId;
+
+import java.util.Date;
+
+@Data
+@RequiredArgsConstructor
+public class VerificationDocument {
```

Comment:
Remember to save the VerificationDocument in case you've changed something (like verified), a function for this purpose would be quite helpfull.